vivo-project/VIVO
# [VIVO-1619] Create weblinks for data properties that have a range of  anyURI and begin with http

[VIVO-1619](https://jira.duraspace.org/browse/VIVO-1619)

# What does this pull request do?
Identifiers that are data properties do not link to a URL even if the range is set to URL.
This change displays the data property value as a weblink IF it is of range URL AND it contains the string http.


# What's new?
Updated the propStatement-dataDefault.ftl file to test the range and value and create a weblink.
All data properties that have a range of URL AND have http in the value will display as a link.
There is no additional verification of the value. This validation should happen upstream in the VIVO ETL or as the user is entering the data.

Example:
* Changes https://github.com/delsborg to a hyperlink

# How should this be tested?
1. Deploy this change
2. Add some data to a data property with a range of anyURI
3. Verify that the data property is now a hyperlink.

# Additional Notes:
This has been tested in CU Boulders development instance.
I would like to see if this can also be applied to a maintenance patch in version 1.9.3 such that we can update our local VIVO repo from that and keep things in sync.

Example:
* Documentation regarding linking to external sties should be updated. It is currently sparse and difficult to identify how to link to external sites.
* Does this change add any new dependencies?  NO
* Does this change require any other modifications to be made to the repository? No
* Could this change impact execution of existing code?  NO
